jobs/bump-lockfile.Jenkinsfile

```diff
@@ -35,10 +35,12 @@ try { lock(resource: "bump-${params.STREAM}") { timeout(time: 120, unit: 'MINUTE
     """)
 
     def branch = params.STREAM
+    def timestampOnly = false
     shwrap("cosa init --branch ${branch} https://github.com/${repo}")
     shwrap("cosa buildprep ${BUILDS_BASE_HTTP_URL}/${branch}/builds")
 
     def prevPkgChecksum = shwrapCapture("jq -c .packages src/config/manifest-lock.*.json | sha256sum")
+    def prevPkgTimestamp = shwrapCapture("jq -c .metadata.generated src/config/manifest-lock.*.json | xargs -I{} date --date={} +%s")
```

Comment:
The logic below assumes that it's a single value, but once we support multi-arch it'll actually be multiple newline-separated values. Let's just hardcode x86_64 for this? Even with multi-arch, I think we should enforce that the top-level `metadata.generated` timestamp be the same to retain our sanity, even if the per-yumrepo timestamps naturally differ.

Reply:
Yeah, we'll need to revisit this soon for multi-arch. Hardcoding to x86_64 for now.